test/com/lynxanalytics/biggraph/frontend_operations/GraphMetricsTest.scala

```diff
@@ -15,7 +15,7 @@ class GraphMetricsTest extends OperationsTestBase {
   }
 
   // All 2 plot tests use this box
-  val plotBox = box("Create example graph")
+  def plotBox = box("Create example graph")
```

Comment:
What is this change for?

Reply:
> What is this change for?

With `val` the right-hand side is executed when the test definitions are loaded. With `def` it's only executed when the tests are running.

`scalatest` loads the test definitions for all tests to see which tests match the requested tags. So this line was triggered no matter which tags we are testing. But when running the RequiresDocker tests (`+backend-test-docker`) I didn't copy in the `built-ins` directory, so this line was causing a failure.

There are two solutions: a) copy in `built-ins` even if we are not running tests that use it, or b) don't touch `built-ins` when no tests running need it. I went with "b)".